rtl/eclrtl/rtldynfield.cpp

```diff
@@ -30,67 +30,83 @@
 
 //---------------------------------------------------------------------------------------------------------------------
 
-const RtlTypeInfo *FieldTypeInfoStruct::createRtlTypeInfo() const
+const RtlTypeInfo *FieldTypeInfoStruct::createRtlTypeInfo(bool useBias) const
 {
     const RtlTypeInfo *ret = nullptr;
-    switch (fieldType & RFTMkind)
+    auto useType = fieldType;
+    if (useBias)
+    {
+#if __BYTE_ORDER == __LITTLE_ENDIAN
```

Comment:
Would it be better to have a RtKeyednteger type (which was derived from RtlIntTypeInfo or RtSwaplIntTypeInfo depending on the endianness), and which implemented the bias support?